chia_streamable_macro/Cargo.toml

```diff
@@ -12,5 +12,5 @@ repository = "https://github.com/Chia-Network/chia_rs/"
 proc-macro = true
 
 [dependencies]
-syn = { version = ">1.0.86", features = ["full"] }
-quote = ">1.0.15"
+syn = { version = "2.0.22", features = ["full"] }
+quote = "1.0.28"
```

Comment:
This is the major version release:
https://github.com/dtolnay/syn/releases/tag/2.0.0

Since it bumps the major version, pinning it to 1.X.X will prevent it from using 2.X.X

Reply:
I would expect that saying `1.0.28` prevents using older versions, such as `1.0.15`. is that not the case? If not, does changing the 15 to 28 here have any effect?

Reply:
It prevents using older versions, yes. But is there a reason not to use syn 2?

Reply:
not that I know of